doc/release_notes.rst

```diff
@@ -117,7 +117,7 @@ Bugs fixed in 4.2.6:
 - Allow globs in setopt in repoid part
 - Fix formatting of message about free space required
 - [doc] Add info of relation update_cache with fill_sack (RhBug:1658694)
-- Fix installation failiure when duplicit RPMs are specified (RhBug:1687286)
+- Fix installation failure when duplicated RPMs are specified (RhBug:1687286)
```

Comment:
This should read _duplicate_, but I'm not actually sure we should be changing release notes for old versions? But if it doesn't hurt anything, why not, at least it won't pop up again in that spellchecking tool.

Reply:
Yeah, I wondered about that as well. It's not like we could time-travel and retrospectively fix such entries, but it also doesn't break anything.

My reasoning would have been that this change helps users searching through the changelog for specific stuff.

The spelling error is a very unfortunate one because it completely changes the entry's meaning (naturally should have been something along the lines duplicate/duplicated, but duplicit sadly *is* a real word and means deceitful) and makes it difficult to be found.

Also ACKing on the duplicated -> duplicate change.